src/xml.tsx

```diff
@@ -171,10 +171,15 @@ export class SvgFromXml extends Component<XmlProps, XmlState> {
   }
 
   parse(xml: string | null) {
+    const { onError = err } = this.props;
     try {
       this.setState({ ast: xml ? parse(xml) : null });
     } catch (e) {
-      console.error(e);
+      const error = e as Error;
+      onError({
+        ...error,
+        message: `Couldn't parse SVG, case: ${error.message}`,
```

Comment:
```suggestion
        message: `[RNSVG] Couldn't parse SVG, reason: ${error.message}`,
```

Would be better maybe?

Reply:
Yep, it looks better, thanks 👍